requirements.txt

```diff
@@ -1,6 +1,6 @@
 beautifulsoup4
-conda-oci-mirror
 deprecated
+git+https://github.com/channel-mirrors/conda-oci-mirror.git@v0.1.0
```

Comment:
This doesn't work because it's passed to micromamba

Reply:
ahh okay